src/test/java/plugins/MavenPluginTest.java

```diff
@@ -246,4 +248,20 @@ private void assertHasModule(MavenModuleSet job, String name) {
         find(by.xpath("//a[@href='%s/']", name)).click();
         assertThat(driver.getCurrentUrl(), equalTo(build.module(name).url.toExternalForm()));
     }
+
+    @Issue("JENKINS-22252")
+    @WithPlugins({"maven-plugin@2.12", "tasks"})
```

Comment:
Shouldn't this be the initial version required to get the test running? I.e., 2.12-SNAPSHOT or 2.13? It is not required to change it when 2.13 has been released. So we can better see in which version the fix has been applied.